ngraph/test/serialize.cpp

```diff
@@ -0,0 +1,932 @@
+//*****************************************************************************
```

Comment:
This file looks redundant. Please, check whether you really need it.

Reply:
removed